dcos-commons-tls-nginx/assets/http.conf

```diff
@@ -0,0 +1,13 @@
+server {
+    listen       80 default_server;
+    listen       [::]:80 default_server;
+    server_name  nginx-https.hello-world.l4lb.thisdcos.directory;
+    root         /usr/share/nginx/html;
+
+    if ($http_x_forwarded_proto = 'http') {
+    return 301 https://$server_name$request_uri;
+    }
+
+    location / {
+    }
+}
```

Comment:
To answer your question its not being used in tests but I see no harm having it in configuration.